ersvr/kaggle_train.py

```
@@ -275,74 +301,88 @@ def _find_sequence_directory(self):
-            has_numbered_dirs = any(item.isdigit() for item in contents if os.path.isdir(os.path.join(directory, item)))
-            has_image_files = any(item.lower().endswith(('.png', '.jpg', '.jpeg')) for item in contents)
+            has_numbered_dirs = any(
+                item.isdigit()
+                for item in contents
+                if os.path.isdir(os.path.join(directory, item))
+            )
+            has_image_files = any(
+                item.lower().endswith((".png", ".jpg", ".jpeg")) for item in contents
+            )
             return has_numbered_dirs or has_image_files
         except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)